### PR TITLE
Will prevent future crashes

### PR DESCRIPTION
**Pull request by aaustin**
@ahmednawar Hot fix for all affected

### CODE REVIEW

**Comment by dmitrig01** [comment]
do we also want to do a read-repair, i.e. clear it if not string?

**Comment by ahmednawar** [comment]
@aaustin @derrickstaten @scott-branchio

File: Branch-TestBed/Branch-TestBed/AppDelegate.m [hunk 1]
```diff
@@ -37,7 +37,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     [branch registerDeepLinkController:controller forKey:@"gravatar_email"];
     
-    [branch initSessionWithLaunchOptions:launchOptions automaticallyDisplayDeepLinkController:YES deepLinkHandler:^(NSDictionary *params, NSError *error) {
+    [branch initSessionWithLaunchOptions:launchOptions isReferrable:YES andRegisterDeepLinkHandler:^(NSDictionary *params, NSError *error) {
```
Comment (derrickstaten) [inline Branch-TestBed/Branch-TestBed/AppDelegate.m]: can we not make this change? it's unrelated. thanks!

**Comment by derrickstaten** [comment]
this looks good to me, and it sounds like you've tested extensively. :+1: after you check the comment above.

File: Branch-TestBed/Branch-TestBed/AppDelegate.m (hunk 1 above)
Comment (ahmednawar) [inline Branch-TestBed/Branch-TestBed/AppDelegate.m]: sure!

**Comment by aaustin** [comment]
:+1:

**Pull request merged**